docs/modules/discovery.md

```diff
@@ -203,6 +203,13 @@ The following list of module parametrization settings can be provided either as
                                       Type: boolean
                                       Default: true
 
+            RejectUnknownRevocationStatus=VALUE
+                                      OPC UA Client Security Config - aReject chain validation 
```

Comment:
`aReject` does not look like a valid word. Maybe the `a` in the font should be removed ?